W-17316096 monitoring/message logging (#783)
mulesoft/docs-gateway: gateway/1.9/modules/ROOT/pages/flex-managed-monitor.adoc

= Monitoring in Managed Flex Gateway
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images

Managed Flex Gateway allows you to access logs and view key metrics.

> Refer to the following tasks for details about monitoring in Connected Mode:

Reviewer: ```suggestion
Refer to these tasks for details about monitoring in Connected Mode:
```
